### PR TITLE
Rewrite activity list for using section list + Add pagination

### PR DESCRIPTION
**Pull request by osdnk**
I see it might be a bit faster, but I'm still not sure about the performance 

(imagine that icons are not dropped)

image

image

### CODE REVIEW

**Comment by osdnk** [comment]
I see it drastically impacted the performance. I'll try to do it a different way

**Comment by osdnk** [comment]
ok, it's good. Still can see whitespaces, but at least it's usable and in prod it should be good.

**Comment by osdnk** [comment]
@skylarbarrera

File: src/hooks/useAccountTransactions.js [hunk 1]
```diff
@@ -17,9 +19,19 @@ export default function useAccountTransactions(initialized, isFocused) {
...
 
+  const [page, setPage] = useState(1);
+  const nextPage = useCallback(() => setPage(page => page + 1), [setPage]);
+
+  const slicedTransaction = useMemo(
+    () => transactions.slice(0, page * NOE_PAGE),
+    [transactions, page]
+  );
+
+  console.log(slicedTransaction.length);
```
Comment (skylarbarrera) [inline src/hooks/useAccountTransactions.js:30]: way want to remove this

**Review by skylarbarrera — APPROVED** [review]
looks good, more performative for sure, gets a little unwieldy when length gets to 90+ but thats the same as the current

File: src/hooks/useAccountTransactions.js (hunk 1 above)
Comment (osdnk) [inline src/hooks/useAccountTransactions.js:30]:
```suggestion
```

**Pull request merged**